transfuse/src/test/java/org/androidtransfuse/gen/FilerSourceCodeWriterTest.java

```diff
@@ -53,14 +53,7 @@ public void setUp() throws Exception {
 
     @Test
     public void testCreateSourceFile() throws IOException {
-
         when(mockFiler.createSourceFile(TEST_PACKAGE + "." + TEST_CLASS)).thenReturn(mockFile);
         when(mockFile.openOutputStream()).thenReturn(mockOutputStream);
-
-        assertEquals(mockOutputStream, codeWriter.openBinary(codeModel._package(TEST_PACKAGE), TEST_CLASS));
-
-        codeWriter.close();
-        verify(mockOutputStream).flush();
-        verify(mockOutputStream).close();
```

Comment:
In general, I'd really like to keep these tests.

Reply:
now it seems this test not doing anything right ? do you have idea what we should test, or should we remove it ?

Reply:
I would breing everything back except for `verify(mockOutputStream).flush();`

Reply:
I just modify the test, and add spy there when creating codeWriter object. Also, i put a new method so we can mock this part
doReturn(mockOutputStream).when(codeWriter).getWriterOutputStream(mockWriter);
please let me know if you have better approach on this.

Reply:
Ok, great, I think that works.